ghcide/src/Development/IDE/Plugin/HLS.hs

```diff
@@ -44,20 +47,48 @@ import           UnliftIO.Exception           (catchAny)
 -- ---------------------------------------------------------------------
 --
 
-data Log
-  = LogNoEnabledPlugins
-  deriving Show
+data Log = LogPluginError ResponseError
+    deriving Show
 
 instance Pretty Log where
   pretty = \case
-    LogNoEnabledPlugins ->
-      "extensibleNotificationPlugins no enabled plugins"
+    LogPluginError err -> responseErrorToLogMessage err
+
+responseErrorToLogMessage :: ResponseError -> Doc a
```

Comment:
`prettyResponseError`? Or even we should have a `Pretty ResponseError` instance somewhere...

Reply:
I think an instance would be best, I'll see if it's possible